setup.py

```diff
@@ -1831,8 +1831,16 @@ def detect_modules(self):
 ##         self.add(Extension('xx', ['xxmodule.c']))
 
         if 'd' not in sysconfig.get_config_var('ABIFLAGS'):
+            # Non-debug mode: Build xxlimited with limited API
             self.add(Extension('xxlimited', ['xxlimited.c'],
-                               define_macros=[('Py_LIMITED_API', '0x03050000')]))
+                               define_macros=[('Py_LIMITED_API', '0x03100000')]))
```

Comment:
No. When new things are needed, we should copy `xxlimited` to e.g. `xxlimited_310`, add tests, and update `xxlimited` to the newest version. If new stuff isn't needed in a release, this can stay on the old limited API version.

Reply:
Ok.